README.md

```diff
@@ -1,102 +1,111 @@
+![Logo](https://raw.githubusercontent.com/datatrails/datatrails-scitt-samples/main/DataTrails_Horizontal_Logo_Black.png)
+![Logo](https://raw.githubusercontent.com/datatrails/datatrails-scitt-samples/main/DataTrails_Horizontal_Logo_White.png)
 
-# datatrails-scitt-samples
+# DataTrails SCITT Examples
 
-The files in this repository can be used to demonstrate how the DataTrails SCITT API works. To do this please download one or more of these files, then follow
+The files in this repository can be used to demonstrate how the DataTrails SCITT API works.
 
-the getting started docs.
+## Copy the Samples
 
-## Generating a signing key
+```shell
+git clone https://github.com/datatrails/datatrails-scitt-samples.git
+
+cd datatrails-scitt-samples
+```
+
+## Generating a Signing Key
 
 In the samples we assume the signed key is an ecdsa p256 key:
 
-```
+```shell
 openssl ecparam -name prime256v1 -genkey -out scitt-signing-key.pem
 ```
 
-## Generating a statement
+## Generating a Statement
 
 In the samples we assume the statement is a json document, e.g:
 
-```
+```shell
+cat > statement.json <<EOF
 {
     "author": "fred",
     "title": "my biography",
     "reviews": "mixed"
 }
+EOF
 ```
 
-## Creating a signed statement
+## Creating a Signed Statement
 
 To create a signed statement we can use a venv.
 
 Create a new venv:
 
-```
-python -m  venv
+```shell
+python -m  venv venv
 ```
 
 Now activate the new venv:
 
-```
+```shell
 source venv/bin/activate
 ```
 
 Now ensure all the requirements are installed:
 
-```
+```shell
+pip install --upgrade pip && \
 pip install -r requirements.txt
 ```
 
 Finally we have an environment we can run the create signed statement script in:
 
-```
-python scitt/create_signed_statement.py --signing-key-file scitt-signing-key.pem \
-    --statement-file scitt-statement.json \
-    --feed testfeed \
-    --issuer testissuer \
-    --output-file signed-statement.txt
+```shell
+python scitt/create_signed_statement.py \
+    --signing-key-file scitt-signing-key.pem \
+    --payload statement.json \
+    --feed my-product-name \
+    --issuer sysnation.dev \
+    --output-file signed-statement.cbor
 ```
 
 Now we have the signed statement we can deactivate the venv:
 
-```
+```shell
 deactivate
 ```
 
 ## Verifying a SCITT counter signed receipt
 
-To verify a countersigned receipt issued from datatrails we can use a venv.
-
+To verify a countersigned receipt issued from DataTrails we can use a venv.
 
 Create a new venv:
 
-```
-python -m  venv
+```shell
+python -m  venv venv
 ```
 
 Now activate the new venv:
 
-```
+```shell
 source venv/bin/activate
 ```
 
 Now ensure all the requirements are installed:
 
-```
+```shell
 pip install -r requirements.txt
 ```
 
 Finally we have an environment we can run the verify counter signed receipt signature script in:
 
-```
+```shell
 python scitt/verify_receipt_signature.py \
-    --scitt-receipt.txt 
+    --receipt-file signed-statement.cbor
```

Comment:
This requires the receipt as a base64 encoded cose sign1 message. Not the raw cbor encoded signed-statement.

Changing back to `scitt-receipt.txt`